src/api.js

```diff
@@ -314,8 +314,13 @@ api.declare({
       let statuses = await instGithub.repos.getStatuses({owner, repo, sha: branch});
       let status = statuses.find(statusObject => statusObject.creator.id === taskclusterBot.id);
 
-      // Then we send a corresponding image.
-      return res.sendFile(status.state + '.svg', fileConfig);
+      if (status) {
+        // If we got a status, send a corresponding image.
+        return res.sendFile(status.state + '.svg', fileConfig);
+      } else {
+        // otherwise, huh, a commit without a status..
+        return res.sendFile('error.svg', fileConfig);
+      }
     } catch (e) {
       if (e.code < 500) {
         return res.sendFile('error.svg', fileConfig);
```

Comment:
I think that might be too broad -- I'd like to get alerted in Sentry to programming errors in this try block.